### PR TITLE
Verbiage changes, assessment fixes, and remove utility from company

### PR DESCRIPTION
**Pull request by RLiNREL**
this connects to #204 #207 #175 

- change the verbiage
- implements the table in #207 (the tag to energy equipment/industrial system not addressed)
- remove utility from company

### CODE REVIEW

File: src/app/company-dashboard/company-settings/company-settings.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h5>
                 <app-location-form [inCompany]="true"></app-location-form>
                 <hr>
                 <label class="bold">Tracked Utilities</label>
-                <app-units-form [inCompany]="true"></app-units-form>
+                <app-units-form></app-units-form>
```
Comment (rmroot) [inline src/app/company-dashboard/company-settings/company-settings.component.html]: We will want to remove this whole section in company-settings related to Tracked Utilities

File: src/app/setup-wizard/data-collection/on-site-assessment/assessment-details-form/assessment-details-form.component.ts [hunk 2]
```diff
@@ -60,6 +66,25 @@ export class AssessmentDetailsFormComponent {
     this.energyEquipmentSub.unsubscribe();
   }
 
+  setUtilityTypes() {
```
Comment (rmroot) [inline src/app/setup-wizard/data-collection/on-site-assessment/assessment-details-form/assessment-details-form.component.ts]: Both of these functions should be tagged `async` and add `await` to the saveChanges calls

`await this.saveChanges()`

File: src/app/setup-wizard/pre-visit/pre-assessment-setup/pre-assessment-setup.component.ts [hunk 3]
```diff
@@ -89,6 +95,26 @@ export class PreAssessmentSetupComponent {
     this.energyEquipmentSub.unsubscribe();
   }
   
+
+  setUtilityTypes(assessment: IdbAssessment) {
```
Comment (rmroot) [inline src/app/setup-wizard/pre-visit/pre-assessment-setup/pre-assessment-setup.component.ts]: same async comment as above.

File: src/app/shared/constants/assessmentTypes.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { UtilityType, UtilityTypes } from "./utilityTypes";
+
+export type AssessmentType = "Pump" | "Fan" | "Process heating" | "Steam" | "Compressed Air" | "Water" | "Treasure Hunt" | "Other";
+export const assessmentTypes: Array<AssessmentType> = ["Pump",  "Fan", "Process heating", "Steam", "Compressed Air", "Water", "Treasure Hunt", "Other"];
+
+// Define the ONE - MANY relationship between Assessment Type and Utility Type
+export interface assessmentOption {
```
Comment (rmroot) [inline src/app/shared/constants/assessmentTypes.ts]: For consistency can we have the constants and types defined starting with capital letters "AssessmentOption" "AssessmentTypes" "AssessmentOptions" please.

Variables start lower case.

File: src/app/shared/helper-pipes/linked-unit-options.pipe.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import { utilityOptions, UtilityType } from '../constants/utilityTypes';
+
+@Pipe({
+  name: 'linkedUnitOptions'
+})
+export class LinkedUnitOptionsPipe implements PipeTransform {
+
+  transform(utilityType: UtilityType): Array<UnitOption> {
+    let selectedUtility = utilityOptions.find(_utilityOption => _utilityOption.utilityType == utilityType);
+    console.log(selectedUtility?.unitOptions.length);
```
Comment (rmroot) [inline src/app/shared/helper-pipes/linked-unit-options.pipe.ts]: remove logging please

**Review by rmroot — CHANGES_REQUESTED** [review]

File: src/app/company-dashboard/company-settings/company-settings.component.html (hunk 1 above)
Comment (RLiNREL) [reply src/app/company-dashboard/company-settings/company-settings.component.html]: good catch

**Review by rmroot — APPROVED** [review]

**Pull request merged**